ethereum_history_api/circuits/lib/src/main.nr

```diff
@@ -19,6 +19,6 @@ fn main(
     state_root: pub [u8; HASH_LENGTH]
 ) -> pub Account {
     let account = verify_account(block_no, address, state_root);
-    verify_header(block_no);
+    let _ = get_header(block_no);
```

Comment:
Not sure we want to do this here. In tests - sure. But here this warning has a reason and with this we can forget about it and just have unused gates in a circuit. This will get solved when this crate becomes a library

Reply:
Agree. `let _` removed